itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/CreateNewProjectTest.java

```diff
@@ -27,36 +36,73 @@
 /**
  * 
  * @author jkopriva@redhat.com
+ * @editor olkornii@redhat.com
  */
 @OpenPerspective(QuarkusPerspective.class)
 @RunWith(RedDeerSuite.class)
 public class CreateNewProjectTest {
 
 	private static String MAVEN_PROJECT_NAME = "testProjectMaven";
-
+	private static String GRADLE_PROJECT_NAME = "testProjectGradle"; 
+	
 	@Test
 	public void testNewNewQuarkusMavenProject() {
 		new WorkbenchShell().setFocus();
 
-		QuarkusWizard qw = new QuarkusWizard();
-		qw.open();
-		assertTrue(qw.isOpen());
+		QuarkusWizard qw = new QuarkusWizard(); // create object QuarkusWizard
+		qw.open(); // opened wizard "New Quarkus Project"
+		assertTrue(qw.isOpen()); // ckeck if passed
+
+		CodeProjectTypeWizardPage wp = new CodeProjectTypeWizardPage(qw); 
+		wp.setProjectName(MAVEN_PROJECT_NAME); // set field "Project name" as "testProjectMaven"
+		wp.setMavenProjectType(); // set field "Project type" as "Maven"
 
-		CodeProjectTypeWizardPage wp = new CodeProjectTypeWizardPage(qw);
-		wp.setProjectName(MAVEN_PROJECT_NAME);
-		wp.setMavenProjectType();
+		qw.next(); // click button "Next"
+		qw.finish(TimePeriod.VERY_LONG); // click button "Finish"
 
-		qw.next();
-		qw.finish(TimePeriod.VERY_LONG);
+		assertTrue(new ProjectExplorer().containsProject(MAVEN_PROJECT_NAME)); // check if testProjectMaven is in Project Explorer
+	
+		checkProblemsView();
+	}
+
+	// https://issues.redhat.com/browse/JBIDE-27057
+	@Test (expected = RedDeerException.class) 
+	public void testNewNewQuarkusGradleProject() {
+		new WorkbenchShell().setFocus();
+
+		QuarkusWizard qw = new QuarkusWizard(); // create object QuarkusWizard
+		qw.open(); // opened wizard "New Quarkus Project"
+		assertTrue(qw.isOpen()); // ckeck if passed
+
+		CodeProjectTypeWizardPage wp = new CodeProjectTypeWizardPage(qw); 
+		wp.setProjectName(GRADLE_PROJECT_NAME); // set field "Project name" as "testGradleProject"
+		wp.setGradleProjectType(); // set field "Project type" as "Gradle"
+
+		qw.next(); // click button "Next"
+		qw.finish(TimePeriod.VERY_LONG); // click button "Finish"
+
+		assertTrue(new ProjectExplorer().containsProject(GRADLE_PROJECT_NAME)); // check if testProjectGradle is in Project Explorer
+		
+		try {
```

Comment:
please format your code with Eclipse default formatter: select code and Ctrl + Shift + F